introduction.md

```diff
@@ -1,2 +1,4 @@
-
-JavaScript
+*git
+*JavaScript
+*html
+*css
```

Comment:
Since you're becoming a GitHub master, could you also add "Killing history using git rebase" as one of your deadly skills?